### PR TITLE
Groups in XCode workspace working.

### PR DESCRIPTION
**Pull request by ricka-github**
I noticed the 'group' directive doesn't do anything for XCode, even though XCode workspaces do support groups.

A nice and simple change to get my feet wet in the codebase.

### CODE REVIEW

**Comment by starkos** [comment]
Looks reasonable to me. Can you add a unit test for it?

**Comment by ricka-github** [comment]
I had to extend the test helper to be able to create a group. It seems to work, but I'm not sure this is the way you want it to be.

File: modules/self-test/test_helpers.lua [hunk 1]
```diff
@@ -44,7 +44,10 @@
 		return prj
 	end
 
-
+	function m.createGroup(wks)		
+		local prj = group ("MyGroup" .. (#wks.groups + 1))
```
Comment (tvandijck) [inline modules/self-test/test_helpers.lua]: just nitpicking I guess... with typically we use 'prj' for projects...
it's a local, so not a big deal, but I would probably rename that local to grp or something.

**Comment by tvandijck** [comment]
Looks good to me... can be merged as is, although I did mark it as 'request changes', it's such a minor thing, that we don't have to hold up the merge for that...

File: modules/self-test/test_helpers.lua (hunk 1 above)
Comment (starkos) [inline modules/self-test/test_helpers.lua]: Technically, we don't need this function at all…you could just use `group()` directly and have full control over the name. But it doesn't hurt to have it for completeness I suppose?

Either way, the change under test is certainly a good one.

**Pull request merged**